docs/index.rst

```diff
@@ -0,0 +1,76 @@
+****************
+MCAP File Format
+****************
+
+MCAP (pronounced "em-cap") is a modular container file format for heterogeneous timestamped data. It is ideal for robotics applications, as it can record multiple streams of structured and unstructured data (e.g. ROS, Protobuf, JSON Schema, etc.) in a single file.
```

Comment:
not clear to the reader how the second part of this sentence supports the first part ("it's ideal for robotics applications"). Maybe the context is that robotics applications involve integrating proprietary stuff that outputs things in formats you don't control? Unclear